Dockerfile

```diff
@@ -1,7 +1,11 @@
-FROM node:14
-WORKDIR /opt/spacebin-api
-COPY package.json yarn.lock ./
-COPY . .
-RUN yarn
+FROM node:latest
```

Comment:
Codacy found an issue: [Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag](https://app.codacy.com/gh/spacebin-org/server/pullRequest?prid=5760887)